CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Version 6.15.0-AF-248 - 2024-10-10
```

Comment:
Typically we do not add pre-release versions to the changelog and we definitely should not have pre-release versions with the same version number